pkg/function-manager/handlers.go

```diff
@@ -123,9 +123,12 @@ func functionModelOntoEntity(m *models.Function, e *functions.Function) error {
 func runModelToEntity(m *models.Run, f *functions.Function) *functions.FnRun {
 	defer trace.Trace("runModelToEntity")()
 	secrets := f.Secrets
-	if m.Secrets != nil && len(m.Secrets) > 0 {
+	if secrets == nil {
 		secrets = m.Secrets
+	} else {
+		secrets = append(secrets, m.Secrets...)
```

Comment:
I was debating with myself whether we should append or replace here. I understand that if function secrets do not overlap with secrets passed to a function run, all of them will be available to the function. If there is an overlap (two secrets with the same key inside), the secret defined in the function run will take precedence, as it has higher slice index and thus will overwrite function default secret. 

If my understanding above is correct, this behavior seems ok.

Reply:
I think your understanding is correct.  Could we add a comment explaining what the intended behavior is?  It's not immediately obvious that the order matters.